systers_portal/templates/meetup/snippets/meetup_sidebar.html

```diff
@@ -0,0 +1,9 @@
+<div class="sidebar-module mb40">
+  <h4>Community Actions</h4>
+  <ol class="list-unstyled">
+    <li><a class="btn btn-info btn-lg btn-block mb15" href="{% url 'about_meetup_location' object.slug%}" role="button">About</a></li>
```

Comment:
Buttons are better looking in a small sidebar in their original size. No need to make them large.
Also if you put the buttons in separate sidebar divs with mb40, then there's no need to add mb15 (margin bottom 15).